demisto_sdk/commands/content_graph/content_graph_commands.py

```diff
@@ -21,32 +31,65 @@ def create_content_graph(
     ContentGraphBuilder(content_graph_interface).create_graph()
     if dependencies:
         content_graph_interface.create_pack_dependencies()
+    if output_path:
+        output_path = output_path / marketplace.value
     if export:
-        content_graph_interface.export_graph()
+        content_graph_interface.export_graph(output_path)
 
 
 def update_content_graph(
     content_graph_interface: ContentGraphInterface,
-    packs_to_update: List[str],
+    marketplace: MarketplaceVersions = MarketplaceVersions.XSOAR,
+    use_git: bool = False,
+    imported_path: Optional[Path] = None,
+    packs_to_update: Optional[List[str]] = None,
     dependencies: bool = True,
+    output_path: Optional[Path] = None,
 ) -> None:
     """This function creates a new content graph database in neo4j from the content path
     Args:
         content_graph_interface (ContentGraphInterface): The content graph interface.
     """
-    ContentGraphBuilder(content_graph_interface).update_graph(packs_to_update)
+    if use_git and imported_path:
+        raise ValueError("Cannot use both git and imported path")
+    if packs_to_update is None and not imported_path:
+        # If no arguments were given, we will use the git diff to get the packs to update
+        use_git = True
+
+    if packs_to_update is None:
+        packs_to_update = []
+    builder = ContentGraphBuilder(content_graph_interface)
+
+    if use_git:
+        get_or_create_graph(content_graph_interface, builder)
+        latest_commit = get_latest_upload_flow_commit_hash()
+        packs_to_update.extend(GitUtil().get_all_changed_pack_ids(latest_commit))
+    else:
+        content_graph_interface.import_graph(imported_path)
+    logger.info(f"Updating the following packs: {packs_to_update}")
+    builder.update_graph(packs_to_update)
     if dependencies:
         content_graph_interface.create_pack_dependencies()
-    content_graph_interface.export_graph()
+    if output_path:
+        output_path = output_path / marketplace.value
+    content_graph_interface.export_graph(output_path)
```

Comment:
what if `output_path` is None? What happens in row 75?

Reply:
No zip file will be creted, and the CSVs of the current graph state will be saved internally in the neo4j folder